src/commands/export.ts

```diff
@@ -6,23 +6,37 @@ import log from '../logger';
 import { isDirectory } from '../utils';
 import setupContext from '../context';
 
-export default async function deploy(params) {
```

Comment:
This function being named `deploy` is a clear mistake, went ahead and changed it to `exportCMD`. Cannot be `export` because it's a reserved keyword.